docs/source/recipes/how-to-use-color-picker-widget.md

```diff
@@ -0,0 +1,139 @@
+---
+myst:
+  html_meta:
+    "description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:title": "How to use the color picker widget"
+    "keywords": "Volto, Plone, frontend, React, blocks, forms, widget, color, picker"
+---
+
+# How to use the color picker widget
+
```

Comment:
I am guessing on the version in which the color picker was added. I think it is 16.0.0 based on this PR: https://github.com/plone/volto/pull/3269

````suggestion

```{versionadded} 16.0.0
```

````

Reply:
IMHO, adding a versionadded is fine for new items, doing it for everything is useless, imagine we had to add in for every single existing feature... Here, this info is not relevant, since for me it's been there forever.

Reply:
I agree that adding a `version*` admonition for anything prior to 16.0.0 and its alphas would be silly at this point. I don't know when the color picker was added, but you make it sound like it's been around long before 16.x.x.

For 16.x.x, only because we still maintain it, I think we should make an effort to include it, but I'll leave it to you as a judgment call.

For 17.x.x, we should definitely make an effort.